3DAmsterdam/Assets/Amsterdam3D/Scripts/Utilities/Editor/MemoryManagement.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+[InitializeOnLoad]
+public class MemoryManagement
```

Comment:
Lege class?

Reply:
Voor nu is die inhoud inderdaad uitgecomment (daarom laat hij dat hier niet zien denk ik..)
Maar wil hem wel graag nog bewaren, omdat ik daar af en toe wat verborgen emscripten variabelen mee aan wil passen.
Voor nu leek default stabieler; maar denk dat ik die in mijn volgende branch nog nodig heb